packages/eslint-plugin/src/rules/keyword-spacing.ts

```diff
@@ -17,7 +17,12 @@ export default util.createRule<Options, MessageIds>({
     },
     fixable: 'whitespace',
     schema: baseRule.meta.schema,
-    messages: baseRule.meta.messages,
+    messages: baseRule.meta.messages ?? {
+      expectedBefore: 'Expected space(s) before "{{value}}".',
+      expectedAfter: 'Expected space(s) after "{{value}}".',
+      unexpectedBefore: 'Unexpected space(s) before "{{value}}".',
+      unexpectedAfter: 'Unexpected space(s) after "{{value}}".',
+    },
```

Comment:
this rule has been using message ids since 5.12.
I guess we should keep this alive until we drop v5.0.0 from the supported ranges.